test_integration/helpers/Erc20TokenHelper.ts

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Please add license at the top.
Rename the file to Eip20TokenHelper.

Reply:
done